keras_cv/layers/preprocessing/random_resized_crop_test.py

```diff
@@ -18,12 +18,26 @@
 
 
 class RandomResizedCropTest(tf.test.TestCase):
-    def test_preserves_output_shape(self):
+    def train_augments_image(self):
```

Comment:
lets add a test case where the area factor is 2.0

Reply:
+1